eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1176,7 +1176,14 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
           // if we are fundee and we were waiting for them to send their first closing_signed, we don't have a lastLocalClosingFee, so we compute a firstClosingFee
           val lastLocalClosingFee = d.closingTxProposed.last.lastOption.map(_.localClosingSigned.feeSatoshis)
           val nextClosingFee = Closing.nextClosingFee(
-            localClosingFee = lastLocalClosingFee.getOrElse(Closing.firstClosingFee(d.commitments, d.localShutdown.scriptPubKey, d.remoteShutdown.scriptPubKey, nodeParams.onChainFeeConf.feeEstimator, nodeParams.onChainFeeConf.feeTargets)),
+            localClosingFee = lastLocalClosingFee.getOrElse {
+              if (d.commitments.localCommit.spec.toLocal == 0.msat) {
```

Comment:
NB: this is only possible for 0-reserve channels, right?

Reply:
It also applies to unused channels with no `push_msat`.